### ISSUE TITLE
Erase?

### ISSUE THREAD

**Issue by beicnet** [igrr/esptool-ck#56]
Using latest bin: esptool-0.4.11-win32.zip

But how I see, I can't erase my WeMos D1 mini

```
esptool.exe -cp COM7 -cb 115200 -cd nodemcu -ce
warning: espcomm_send_command: didn't receive command response
error: espcomm_erase_flash: FLASH_DOWNLOAD_BEGIN failed
error: Invalid argument or value after -ce (argument #8)
```

btw...but I can flash it without any issue.

**Comment by nissimzur** [comment]
I believe you can’t flash it too, based on the log you send

 

 

________

Kind regards,
Dr. Nissim Zur
Wireless long range IoT Artificial Intelligence platforms
Wi-Fi, Bluetooth Smart, Sub GHz
Consultant & Lecturer, 
Research IoT Projects


Tel          +972 544 500635 Israel
Tel          +972 3 5716699
Skype      <callto:nissim.test> nissim.test
Email       <mailto:nissim@telnt.com> nissim@telnt.com

web          <http://www.telnt.com/> www.telnt.com

Zoom       <https://zoom.us/download> https://zoom.us/download  

Zoom ID  <https://zoom.us/j/3717699114> https://zoom.us/j/3717699114

**Comment by beicnet** [comment]
@nissimzur Yes, I can, I flashed it a few times yesterday and today too, but can't erase, I thinks its related to timing.

**Comment by nissimzur** [comment]
Just send a bin file that has 0xFFFF all over.

www.telnt.com

**Comment by pieman64** [comment]
I also found out that I can't erase but "flashing" is working of sorts.
The flash is accepted but the ESP doesn't seem to reboot into running mode. Will do further tests.
What I did find I was able to flash a general sketch to a "dead" ESP01 and then return to the Arduino IDE and now it's running with an OTA hostname of Lazarus. I have a few dead ESP07's and at least one dead WeMos so I will try to resurrect them too when I have time.

**Comment by beicnet** [comment]
I figured out, that it will erase, but also will freeze the command prompt window for a while, is it a bug?

**Comment by pieman64** [comment]
Tried again today and erasing flash still doesn't work. Argument #8 error as per @beicnet 
Anyone have a real fix?

**Comment by beicnet** [comment]
@pieman64 Unfortunately, there is no explanation given for that issue yet.

**Comment by nissimzur** [comment]
The chip does not "know" if it flashes a bin file filled of 0xFFFF or a code. 
So, if you can flash 0XFFFF ok or other data, it must flash good apps bin file. 
If it does not run, you have a bug in your bin. 
Call my SKype nissmi.test I will help

**Comment by beicnet** [comment]
@nissimzur We are talking here about "erasing" chip with command line prompt
